src/context.cpp

```diff
@@ -87,8 +87,9 @@ namespace Sass {
 
   {
 
-    // add cwd to include paths
-    include_paths.push_back(CWD);
+    // Sass 3.4: The current working directory will no longer be placed onto the Sass load path by default.
```

Comment:
Not sure. I was under the impression current master is going for 3.5. Side note: we could use a runtime test for 3.4 vs 3.5 behavior, but a compiler directive is not simple, since we have a string in that constant (string compare is not really supported by `#if` preprocessor directives).